celo-sepolia.env

```diff
@@ -0,0 +1,139 @@
+
+# Set to "full" to force op-geth to use --syncmode=full
+OP_GETH__SYNCMODE=
+
+IS_CUSTOM_CHAIN=true
+
+# Path to the datadir, If the datadir is empty then a new datadir will be
+# initialised at the given path.
+#
+# The path can be absolute, or relative to the docker-compose.yml file.
+DATADIR_PATH=./envs/${NETWORK_NAME}/datadir
+
+# If the datadir is on a disk that doesn't support unix domain sockets then you
+# will need to specify a path to a disk that does support unix domain sockets.
+# E.g. your normal hard drive. If left unset the default path inside the
+# datadir will be used.
+IPC_PATH=
+
+# Controls how op-geth determines its public IP that is shared via the
+# discovery mechanism. The value should be one of
+# (any|none|upnp|pmp|pmp:<IP>|extip:<IP>|stun:<IP:PORT>) if any is selected
+# op-geth will try to automatically determine its external IP. To explicitly
+# set the IP that op-geth can be reached on use extip:<your-external-ip>. To
+# check the value that op-geth is currently using look in the op-geth logs for
+# an entry such as
+# self=enode://b24c34e53adc6db27fe648615eca3b9062a58295242caeea1604690507d507d0138ddeecb3cc9805a1cbd441471790c66a6f886f0538c3cde3e0b5dbd145f1f1@127.0.0.1:30303
+# alternatively you can use the admin_nodeInfo RPC to query this information.
+# If unset other nodes will not be then other nodes on the network will not be
+# able to discover and connect to your node.
+OP_GETH__NAT=any
+
+# This controls the IP that op-node shares with the network so that other nodes may discover and connect to it.
+# To check the value that op-node is currently using you can look in the logs for an entry such as:
+# msg="started p2p host" addrs="[/ip4/127.0.0.1/tcp/9222 /ip4/192.168.97.9/tcp/9222]" peerID=16Uiu2HAkv7PQ5hpa2HeWgjYQ7SChvipCWm3L95hUKejSKMM4rVPe
+# alternatively you can use the op-node opp2p_self RPC to query this
+# information. If unset other nodes will not be then other nodes on the network
+# will not be able to discover and connect to your node.
+OP_NODE__P2P_ADVERTISE_IP=
+
+###############################################################################
+#                            ↓ REQUIRED (EIGENDA) ↓                           #
+###############################################################################
+
+# Specifies the endpoint of the eigenda proxy to use. If this is unset then a local eigenda proxy will be used.
+EIGENDA_PROXY_ENDPOINT=
+
+EIGENDA_LOCAL_SVC_MANAGER_ADDR=0xD4A7E1Bd8015057293f0D0A557088c286942e84b
+EIGENDA_LOCAL_DISPERSER_RPC=disperser-holesky.eigenda.xyz:443
+EIGENDA_LOCAL_SIGNER_PRIVATE_KEY_HEX=
+EIGENDA_V2_LOCAL_SVC_MANAGER_ADDR=0xD4A7E1Bd8015057293f0D0A557088c286942e84b
+EIGENDA_V2_LOCAL_DISPERSER_RPC=disperser-holesky.eigenda.xyz:443
+EIGENDA_V2_LOCAL_SIGNER_PAYMENT_KEY_HEX=
+EIGENDA_V2_LOCAL_CERT_VERIFIER_ADDR=0xFe52fE1940858DCb6e12153E2104aD0fDFbE1162
+EIGENDA_V2_LOCAL_BLS_OPERATOR_STATE_RETRIEVER_ADDR=0xB4baAfee917fb4449f5ec64804217bccE9f46C67
```

Comment:
@gastonponti @piersy Where do I get this info from?

Reply:
Just saw this, sorry.
Before they have that in the eigen docs explicitly. Right now they ask you to retrieve it from their `EigenDADirectory` contract.
The directory address and the disperser url:
https://docs.eigencloud.xyz/products/eigenda/networks/sepolia
The actual directory that should be read as a proxy:
https://sepolia.etherscan.io/address/0x9620dC4B3564198554e4D2b06dEFB7A369D90257#readProxyContract